numpy/core/tests/test_regression.py

```diff
@@ -2115,6 +2115,10 @@ def __array__(self,*args,**kwargs):
         for f in [op.lt, op.le, op.gt, op.ge]:
             if sys.version_info[0] >= 3:
                 assert_raises(TypeError, f, lhs, rhs)
+            elif sys.version_info[0] < 3 and sys.py3kwarning:
+                warnings.filterwarnings('error',category=DeprecationWarning)
```

Comment:
Sorry but could you point out to some locations? I could not spot violations (4 spacing rules, max line width, etc are followed)